exercises/nucleotide-count/tests/nucleotide-count.rs

```diff
@@ -17,19 +17,31 @@ fn check_dna(s: &str, pairs: &[(char, usize)]) {
 
 #[test]
 fn test_count_empty() {
-    assert_eq!(dna::count('A', ""), 0);
+    assert_eq!(dna::count('A', "").unwrap(), 0);
```

Comment:
should we do anything specific here to make abundandly clear that it's a result? see https://github.com/exercism/xrust/blob/master/exercises/largest-series-product/tests/largest-series-product.rs#L6